frontend/src/scenes/authentication/login2FALogic.ts

```diff
@@ -66,7 +66,7 @@ export const login2FALogic = kea<login2FALogicType>([
                     : null,
             }),
             submit: async ({ token }, breakpoint) => {
-                await breakpoint()
+                breakpoint()
```

Comment:
These ones confuse me totally as I thought the whole thing that started off the linting exercise was realizing we weren't awaiting the breakpoints?

Reply:
The `await` is only needed if you pass in an actual numeric value to wait. Otherwise the code executes synchronously and the `await` isn't needed. It _should_ be just a no-op then and won't hurt, but it's effectively ignored.

Essentially:

```ts
function breakPoint(delay?: ms): void | Promise<void> {
  if (delay > 0) { 
    await sleep(delay)
  }
  if (actionWasCalledAgain()) {
    throw BreakpointException();
  }
}
```